test/e2e/apps/daemon_restart.go

```diff
@@ -106,17 +107,20 @@ func (r *RestartDaemonConfig) waitUp(ctx context.Context) {
 	var healthzCheck string
 	if r.enableHTTPS {
 		healthzCheck = fmt.Sprintf(
-			"curl -sk -o %v -I -w \"%%{http_code}\" https://localhost:%v/healthz", nullDev, r.healthzPort)
+			"curl -sk -o %v -I -w \"%%{http_code}\" https://127.0.0.1:%v/healthz", nullDev, r.healthzPort)
```

Comment:
this only works for IPv4, any reason to replace this?

Reply:
@aojea sorry this sneaked in, will file a PR to revert this line

Reply:
no worries, I submitted another PR https://github.com/kubernetes/kubernetes/pull/127070